src/components/Checkout/Checkout.tsx

```diff
@@ -59,15 +46,16 @@ export const CheckoutProcess: React.FC<CheckoutProcessProps> = ({ onClose }) =>
       </button>
       <h1 className={styles.title}>Payment & Delivery</h1>
       <form onSubmit={handleSubmit} className={styles.form}>
-        {formFieldsData.slice(0, 5).map(({ id, type, placeholder, pattern }) => (
+        {formFieldsData.slice(0, 5).map(({ id, type, label, pattern, placeholder }) => (
```

Comment:
it's a duplicating of a code, you can create an independent component for form inputs